NuGet.config

```diff
@@ -11,6 +11,7 @@
     <add key="roslyn-tools" value="https://dotnet.myget.org/F/roslyn-tools/api/v3/index.json" />
     <!-- Used for the SiteExtension 3.1 bits that are included in the 5.0 build -->
     <add key="dotnet31-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet3.1-transport/nuget/v3/index.json" />
+    <add key="general-testing" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/general-testing/nuget/v3/index.json" />
```

Comment:
I added this feed for testing. The main issue was that WiX doesn't have a package for the toolset that supports ARM64. Other repositories don't use `wixproj`, so we didn't need it before this. I don't know what feed this should go to, or if this is even a good idea.

Reply:
Yeah, this was to verify this won't break in CI. I'm asking for some guidance here.

Reply:
@dougbu I got the green light to move this as is to `dotnet-eng`. I've reverted all changes to `NuGet.config`